index.html

```diff
@@ -38,147 +45,183 @@ <h2 class="headline-heading">I'm Suthish Prasanna<br>Glad to see you!</h2>
-        A daily selection of privately personalized reads; no accounts or sign-ups required.
-    </p>
-    <ul class="works-card-tags">
-        <li class="works-card-tag">html</li>
-        <li class="works-card-tag">css</li>
-        <li class="works-card-tag">javascript</li>
+<section class="works">
+    <!-- Card 1 starts -->
+    <ul class="works-cards">
+        <li class="works-card">
+            <img src="./assets/Snapshoot Portfolio (1).png" alt="Recent Work">
+            <div class="works-card-main">
+                <h2 class="works-card-heading-large">Tonic</h2>
+                <div class="works-card-client">
+                    <p class="paragraph">CANOPY</p>
+                    <p class="fa-solid fa-circle works-card-client-counter"></p>
+                    <p class="works-card-client-role paragraph">Back End Dev</p>
+                    <p class="fa-solid fa-circle works-card-client-counter"></p>
+                    <p class="works-card-client-year paragraph">2015</p>
+                </div>
+                <p class="paragraph works-card-paragraph">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul class="works-card-tags">
+                    <li class="works-card-tag">html</li>
+                    <li class="works-card-tag">css</li>
+                    <li class="works-card-tag">javascript</li>
+                </ul>
+                <button>See Project</button>
+            </div>
+        </li>
+        <!-- Card 1 Ends -->
+        <!-- Card 2 starts -->
+        <li class="works-card">
+            <img src="./assets/Snapshoot Portfolio (3).png" alt="Recent Work">
+            <div class="works-card-main">
+                <h2 class="works-card-heading-large">Multi-Post Stories</h2>
+                <div class="works-card-client">
+                    <p class="paragraph">FACEBOOK</p>
+                    <p class="fa-solid fa-circle works-card-client-counter"></p>
+                    <p class="works-card-client-role paragraph">Full Stack Dev</p>
+                    <p class="fa-solid fa-circle works-card-client-counter"></p>
+                    <p class="works-card-client-year paragraph">2015</p>
+                </div>
+                <p class="paragraph works-card-paragraph">
+                    Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.
+                </p>
+                <ul class="works-card-tags">
+                    <li class="works-card-tag">html</li>
+                    <li class="works-card-tag">Ruby on rails</li>
+                    <li class="works-card-tag">css</li>
+                    <li class="works-card-tag">javascript</li>
+                </ul>
+                <button>See Project</button>
+            </div>
+        </li>
+        <!-- Card 2 Ends -->
```

Comment:
I think you forgot to implement the comment given by last reviewer about spacing out the elements vertically. 

| Your Design   | FIGMA Design  |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/61361037/221348177-9c948cef-f3dc-47d9-a7f1-83ac5591baf6.png)| ![image](https://user-images.githubusercontent.com/61361037/221348140-e53e29a9-b97a-440d-819c-dfc63b4d6dad.png)|